src/main/java/seedu/address/model/person/Organ.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's organ in ORGANice.
+ * Guarantees: immutable; is valid as declared in {@link #isValidOrgan(String)}
+ */
+public class Organ {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Please key in a valid organ. Valid organ(s): kidney.";
+
+    public static final String VALIDATION_REGEX = "[^\\s].*";
```

Comment:
seems redundant to have validation regex here

Reply:
Removed as well.